src/main/java/me/programmers/springboot/basic/springbootbasic/command/voucher/CreateVoucherCommand.java

```diff
@@ -0,0 +1,33 @@
+package me.programmers.springboot.basic.springbootbasic.command.voucher;
+
+import me.programmers.springboot.basic.springbootbasic.command.CommandStrategy;
+import me.programmers.springboot.basic.springbootbasic.io.ConsoleInput;
+import me.programmers.springboot.basic.springbootbasic.io.In;
+import me.programmers.springboot.basic.springbootbasic.voucher.VoucherType;
+import me.programmers.springboot.basic.springbootbasic.voucher.model.Voucher;
+import me.programmers.springboot.basic.springbootbasic.voucher.service.JdbcVoucherService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CreateVoucherCommand implements CommandStrategy {
+
+    private static final Logger logger = LoggerFactory.getLogger(CommandStrategy.class);
+    private final ConsoleInput console = new In();
+    private final JdbcVoucherService voucherService;

```

Comment:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(CommandStrategy.class);
    
    private final ConsoleInput console = new In();
    private final JdbcVoucherService voucherService;

```

상수와 필드는 개행으로 구분해주는게 가독성이 더 높습니다.